sunkit_dem/base_model.py

```diff
@@ -75,6 +75,11 @@ def _keys(self):
     def temperature_bin_centers(self) -> u.K:
         return (self.temperature_bin_edges[1:] + self.temperature_bin_edges[:-1])/2
 
+    @property
+    @u.quantity_input
```

Comment:
Still don't understand why this is needed.

Reply:
Without it there's no validation on the output. I guess it isn't strictly needed since the property it is derived from is already checked.